server/src/main/java/com/objectcomputing/checkins/services/feedback_request/FeedbackRequestController.java

```diff
@@ -79,11 +79,9 @@ public FeedbackRequestController(FeedbackRequestServices feedbackReqServices,
      * @return {@link HttpResponse}
      */
     @Delete("/{id}")
-    public Single<HttpResponse> delete(@NotNull UUID id) {
-        return Single.fromCallable(() -> feedbackReqServices.delete(id))
-                .observeOn(Schedulers.from(eventLoopGroup))
-                .map(successFlag -> (HttpResponse) HttpResponse.ok())
-                .subscribeOn(Schedulers.from(executorService));
+    public HttpResponse<?> delete(@NotNull UUID id) {
+        feedbackReqServices.delete(id);
+        return HttpResponse.ok();
```

Comment:
Why are you removing the reactive bits? This has a significant effect and isn't advisable. Were you running into problems?